app/experimenter/nimbus-ui/schema.graphql

```diff
@@ -140,6 +140,7 @@ type NimbusDocumentationLinkType {
 enum NimbusExperimentApplication {
   DESKTOP
   FENIX
+  IOS
```

Comment:
Nice refactor, minimal impact into the GQL boundary, 👍

Reply:
😈 😈 😈 😈